custom_components/tuya_local/devices/ferroli_titano_twin.yaml

```diff
@@ -0,0 +1,64 @@
+name: Ferroli Titano Twin
+products:
+  - id: lci0ebsz7ftaruaf
+    name: DT EWH
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
```

Comment:
for water_heater, this should become operation_mode, and I suggest mapping manual to electric and eco to eco.